types/react-relay/index.d.ts

```diff
@@ -25,15 +26,22 @@ import * as RelayRuntimeTypes from "relay-runtime";
 
 // Taken from https://github.com/pelotom/type-zoo
 // tslint:disable-next-line:strict-export-declare-modifiers
-type Omit<T, K extends keyof T> = Pick<T, ({ [P in keyof T]: P } & { [P in K]: never } & { [x: string]: never, [x: number]: never })[keyof T]>;
+type Omit<T, K extends keyof T> = Pick<
+    T,
+    ({ [P in keyof T]: P } & { [P in K]: never } & { [x: string]: never; [x: number]: never })[keyof T]
+>;
```

Comment:
Why not use `Pick<T, Exclude<keyof T, K>>`?

Reply:
I was actually doing that right now indeed 👍